src/Application/Common/Behaviours/UnhandledExceptionBehaviour.cs

```
@@ -23,7 +23,14 @@ public async Task<TResponse> Handle(TRequest request, RequestHandlerDelegate<TRe
         {
             var requestName = typeof(TRequest).Name;
 
-            _logger.LogError(ex, "OfficeEntry Request: Unhandled Exception for Request {Name} {@Request}", requestName, request);
+            if (ex.Message?.StartsWith("Unhandled exception in circuit") ?? false)
```

@GabrielFG We could possibly just catch "System.Threading.Tasks.TaskCanceledException" (optionnaly with a when clause that checks the message) instead and perhaps swallow the exception?

---

Change has been done.